### DISCUSSION TITLE
Howto stop Repeat Node

### DISCUSSION THREAD

**Discussion by jheisel01** [jensrossbach/node-red-contrib-chronos#59] [Q&A] [answered]
Hi,
some questions about the repeat node.

Attention, I am still new in the NodeRed environment 
a) Can I start the repeat node with a msg ?
b) How can I stop the repeat with a msg ?

e.g. for stop msg.topic = "abc" , payload ="stop".

regards,
Juergen

**Comment by jensrossbach** [comment, marked as answer]
Hi!

> a) Can I start the repeat node with a msg ?

The repeat node starts sending messages as soon as it receives an input message (this input message is then periodically repeated).

> b) How can I stop the repeat with a msg ?

Repetition stops as soon as another message is received (and repetition of that message begins). You can also specify an ending time for repetition in the node's configuration. And a third option is to send a message to the node which contains a property with the name "stop" (the type of that property does not matter). Such message immediately stops repetition of the previous message and the node becomes idle.

You can find more information in the wiki:
https://github.com/jensrossbach/node-red-contrib-chronos/wiki/Repeat-Node
